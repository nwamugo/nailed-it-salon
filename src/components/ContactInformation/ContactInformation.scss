.contact-information .primary-header {
  color: #e294c5;
  text-align: center;
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 4.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  margin-top: 4.6rem;
}

.contact-information .primary-location {
  text-align: center;
  font-family: "Roboto", sans-serif;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #fff;
  margin-top: 2.8rem;
}
.contact-information .primary-info {
  text-align: center;
  font-family: "Roboto", sans-serif;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #fff;
}

.contact-information .secondary-header {
  text-align: center;
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 3rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  margin-top: 4.6rem;
  color: #f8e7f2;
}
