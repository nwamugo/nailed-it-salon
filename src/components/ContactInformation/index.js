import React from 'react';
import './ContactInformation.scss';

const ContactInformation = () => {
  return (
    <div className="contact-information">
      <h2 className="primary-header">Contact Us</h2>
      <h3 className="primary-location">
        Ground Foor, Al Raed Groove, Ashirat Sudayr street, &nbsp;
        <br />
        Ar Raid, Riyadh 12355
      </h3>
      <h3 className="primary-info">
        info@mysite.com&nbsp;&nbsp;|&nbsp;&nbsp;+966 56 057 7709
      </h3>
      <h2 className="secondary-header">Opening Hours</h2>
      <h3 className="primary-info">
        Mon - Thur &amp; Sat: 11am - 9pm
        <br />
        Friday: 2pm - 9pm
        <br />
        Sunday: Closed
      </h3>
    </div>
  );
};

export default ContactInformation;
