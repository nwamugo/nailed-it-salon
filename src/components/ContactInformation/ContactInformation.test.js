import React from 'react';
import { shallow } from 'enzyme';
import ContactInformation from './index';

describe('ContactSection', () => {
  it('should not have regressed', () => {
    const component = shallow(<ContactInformation />);

    expect(component).toMatchSnapshot();
  });
});
