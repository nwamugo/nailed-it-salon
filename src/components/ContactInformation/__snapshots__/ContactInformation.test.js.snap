// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection should not have regressed 1`] = `
<div
  className="contact-information"
>
  <h2
    className="primary-header"
  >
    Contact Us
  </h2>
  <h3
    className="primary-location"
  >
    Ground Foor, Al Raed Groove, Ashirat Sudayr street,  
    <br />
    Ar Raid, Riyadh 12355
  </h3>
  <h3
    className="primary-info"
  >
    info@mysite.com  |  +966 56 057 7709
  </h3>
  <h2
    className="secondary-header"
  >
    Opening Hours
  </h2>
  <h3
    className="primary-info"
  >
    Mon - Thur & Sat: 11am - 9pm
    <br />
    Friday: 2pm - 9pm
    <br />
    Sunday: Closed
  </h3>
</div>
`;
